.collection/README.md

```diff
@@ -9,6 +9,8 @@ This collection contains a role for managing Microsoft SQL Server.
 * Red Hat Enterprise Linux 9 (RHEL 9+)
 * Fedora
 * RHEL derivatives such as CentOS
+* Suse Linux Enterprise Server 15 (SLES 15)
```

Comment:
Please rename the subtitle above from "Currently supported distributions" to "Compatible OS" to be more specific.

Reply:
I am stealing this smart approach from https://github.com/linux-system-roles/ha_cluster/pull/188